library/include/hipblas.h

```diff
@@ -13623,18 +14458,77 @@ HIPBLAS_EXPORT hipblasStatus_t hipblasCgeqrfBatched(hipblasHandle_t       handle
                                                     const int             lda,
                                                     hipblasComplex* const ipiv[],
                                                     int*                  info,
-                                                    const int             batch_count);
+                                                    const int             batchCount);
 
+/*! \brief SOLVER API
+
+    \details
+    geqrfBatched computes the QR factorization of a batch of general
+    m-by-n matrices.
+
+    (This is the blocked version of the algorithm).
+
+    The factorization of matrix \f$A_j\f$ in the batch has the form
+
+    \f[
+        A_j = Q_j\left[\begin{array}{c}
+        R_j\\
+        0
+        \end{array}\right]
+    \f]
+
+    where \f$R_j\f$ is upper triangular (upper trapezoidal if m < n), and \f$Q_j\f$ is
+    a m-by-m orthogonal/unitary matrix represented as the product of Householder matrices
+
+    \f[
+        Q_j = H_{j_1}H_{j_2}\cdots H_{j_k}, \quad \text{with} \: k = \text{min}(m,n)
+    \f]
+
+    Each Householder matrix \f$H_{j_i}\f$ is given by
+
+    \f[
+        H_{j_i} = I - \text{ipiv}_j[i] \cdot v_{j_i} v_{j_i}'
+    \f]
+
+    where the first i-1 elements of Householder vector \f$v_{j_i}\f$ are zero, and \f$v_{j_i}[i] = 1\f$.
+
+    @param[in]
+    handle    hipblasHandle_t.
+    @param[in]
+    m         int. m >= 0.\n
+              The number of rows of all the matrices A_j in the batch.
+    @param[in]
+    n         int. n >= 0.\n
+              The number of columns of all the matrices A_j in the batch.
+    @param[inout]
+    A         Array of pointers to type. Each pointer points to an array on the GPU of dimension lda*n.\n
+              On entry, the m-by-n matrices A_j to be factored.
+              On exit, the elements on and above the diagonal contain the
+              factor R_j. The elements below the diagonal are the last m - i elements
+              of Householder vector v_(j_i).
+    @param[in]
+    lda       int. lda >= m.\n
+              Specifies the leading dimension of matrices A_j.
+    @param[out]
+    ipiv      pointer to type. Array on the GPU.\n
```

Comment:
Array of pointers to type. Each pointer points to an array on the GPU of dimension min(m, n).